crates/sui-e2e-tests/tests/traffic_control_tests.rs

```diff
@@ -738,6 +824,7 @@ async fn test_traffic_sketch_allowlist_mode() {
 }
 
 async fn assert_traffic_control_ok(mut test_cluster: TestCluster) -> Result<(), anyhow::Error> {
+    telemetry_subscribers::init_for_testing();
```

Comment:
what's the reason for adding this to the different tests?

Reply:
without it,  you won't get logs printed when you run unit tests